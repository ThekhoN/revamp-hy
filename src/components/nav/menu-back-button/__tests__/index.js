import React from 'react';
import renderer from 'react-test-renderer';
import MenuBackButton from '../index';

describe('MenuBackButton', () => {
  it ('renders correctly', () => {
    const tree = renderer.create(
      <MenuBackButton />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
