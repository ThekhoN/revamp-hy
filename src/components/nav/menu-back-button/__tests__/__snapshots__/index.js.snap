// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuBackButton renders correctly 1`] = `
<div
  className="mobile-ui--only pos--absolute"
  onClick={undefined}
  style={
    Object {
      "left": "0px",
      "top": "0px",
    }
  }
>
  <span
    className="svg-icon-wrapper"
  >
    <svg
      aria-labelledby="title"
      height="48"
      viewBox="0 0 48 48"
      width="48"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title
        id="title"
      >
        Homeyantra Navigation Close
      </title>
      <path
        d="M22.7 24l4.7-4.7c.2-.2.3-.4.3-.7 0-.3-.1-.5-.3-.7-.2-.2-.4-.3-.7-.3-.3 0-.5.1-.7.3l-5.4 5.4c-.2.2-.3.4-.3.7s.1.5.3.7l5.4 5.4c.2.2.4.3.7.3.3 0 .5-.1.7-.3.2-.2.3-.4.3-.7 0-.3-.1-.5-.3-.7L22.7 24z"
        fill="#38afc7"
      />
    </svg>
  </span>
</div>
`;
